elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -141,7 +141,12 @@ $variables: (
       align-items: center;
       height: 100%;
     }
-
+    // Default CTA arrow wrap fix for Firefox
+    :host( :not([pfe-priority]):not([aria-disabled="true"])) & {
+      @include browser-query(firefox) {
+        max-width: calc(100% - 8px - #{pfe-local($cssvar: size, $region: arrow)});
```

Comment:
Where does the 8px come from?

Reply:
it represents the width of the `&#160;` character

Reply:
Ah, I think `1ch` will serve this better then because if you change font size, the non breaking space size changes.

Reply:
LOL, I promise I actually tried this before, and I thought it didn't work. However I think the issue I was seeing was a `<button>` inside the `<pfe-cta>` (or some other issue) and I falsely attributed it to the `ch`. Anyway it seems to work just fine. Gonna push up that change.

Reply:
For posterity, I re-tested the `ch` in Firefox and IE11 and iphone 10